src/main.cpp

```diff
@@ -1,74 +1,57 @@
 #include <iostream>
+#include <vector>
+
 #include <GL/glew.h>
 #include <glm/glm.hpp>
-#include "display.h"
-#include "mesh.h"
-#include "shader.h"
-#include "transform.h"
-#include "objectLoader.h"
-#include <vector>
 
-float getGreatestValue(glm::vec3 * verticesArray, unsigned int size);
+#include "Display.h"
+#include "Mesh.h"
+#include "ObjectLoader.h"
+#include "Shader.h"
+#include "Transform.h"
+
+float getGreatestValue(glm::vec3* verticesArray, unsigned int size);
 
 int main(int argc, char **argv)
 {
 	Display display (800, 600, "Solar Simulator");
 
-	/*glm::vec3 verticesArray[] = { (glm::vec3(-0.5, -0.5, 0)), //OpenGL coordinates are from -1 to 1
-						 (glm::vec3(0, 0.5, 0)),
-						 (glm::vec3(0, -0.5, 0)),
-						(glm::vec3(-0.5, -0.5, 0.5)),
-						(glm::vec3(0, 0.5, 0.5)),
-						(glm::vec3(0.5, -0.5, 0.5))};
-
-	unsigned int indices[] = { 0, 1 , 2, 3, 4, 5};*/
 	std::vector<glm::vec3> vertices;
 	std::vector<glm::vec3> normals; // Won't be used at the moment.
 	unsigned int numVertices = loadObject("test2.obj", vertices, normals);
-	glm::vec3 * verticesArray = &vertices[0];
-	unsigned int * indices = new unsigned int[numVertices];
+	glm::vec3* verticesArray = &vertices[0];
+	unsigned int* indices = new unsigned int[numVertices];
```

Comment:
Fix ownership of "indices". Currently it is a memory leak. Why not use a std::vector or a shared_pointer depending on the application?